pages/api/heroes/[heroId].js

```diff
@@ -0,0 +1,30 @@
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+export default async function handler(req, res) {
+  const { heroId } = req.query;
+  if (req.method === 'GET') {
+    // GET METHOD
+    const heroData = await prisma.heroes.findUnique({
+      where: {
+        id: parseInt(heroId),
+      },
+    });
+    res.status(200).json(heroData);
+  } else if (req.method === 'PATCH') {
+    // PATCH METHOD
+    const updatedData = req.body;
+    try {
+      const Heroes = await prisma.heroes.update({
+        where: {
+          id: parseInt(heroId),
+        },
+        data: updatedData,
+      });
+    } catch (error) {
+      console.log(error.meta.cause);
```

Comment:
```suggestion
      res.status(500).json(error.meta.cause)
```

ini case kalo si server kita ngalamin error, harus return `json` juga yak mas @berry1611

Reply:
kalo untuk json nya isinya tetap error.meta.cause kah? ato message bebas aja? @ariefkahfi